libsrc/leddevice/dev_net/LedDeviceYeelight.cpp

```diff
@@ -0,0 +1,1503 @@
+﻿#include "LedDeviceYeelight.h"
+
+// ssdp discover
+#include <ssdp/SSDPDiscover.h>
+
+// Qt includes
+#include <QEventLoop>
+
+#include <QtNetwork>
+#include <QTcpServer>
+#include <QColor>
+#include <QDateTime>
+
+#include <chrono>
+#include <thread>
+
+static const bool verbose  = false;
+static const bool verbose3  = false;
+
+// Constants
+const int WRITE_TIMEOUT	= 1000;			 // device write timeout in ms
+const int READ_TIMEOUT	= 1000;			 // device write timeout in ms
+const int CONNECT_TIMEOUT = 1000;		 // device connect timeout in ms
+const int CONNECT_STREAM_TIMEOUT = 1000; // device streaming connect timeout in ms
+
+static const bool TEST_CORRELATION_IDS  = false; //Ignore, if yeelight sends responses in different order as request commands
```

Comment:
Instead of marking all those variables as static, what about putting them in an anonymous namespace ? It will do the same job. Using static keyword in this context is more of a C style.

Reply:
"Using static keyword in this context is more of a C style."
I though not using #define, but having even types is more c++, but if namespace, if the way to go, happy to have a look at.

Reply:
It has some advantages too : https://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static

Reply:
Does this mean we should review the static functions, too?
e.g. 
LedDeviceWrapper
	static const QJsonObject getLedDeviceSchemas();

Reply:
Updates done on all LedDevice files

Reply:
> Does this mean we should review the static functions, too?
> e.g.
> LedDeviceWrapper
> static const QJsonObject getLedDeviceSchemas();

maybe with the time...